kubernetes/samples/scripts/create-oracle-db-service/common/oracle.db.yaml

```diff
@@ -71,4 +71,4 @@ spec:
       securityContext: {}
```

Comment:
I checked this file in accidentally and will remove it.  It's implicitly updated when you start the DB using the DB sample...